doc/contributing/releases.md

```diff
@@ -282,7 +282,13 @@ You can integrate the PRs into the proposal without running full CI.
 
 ### 2. Create a new branch for the release
 
-⚠️ At this point, you can either run `git node release --prepare`:
+> \[!TIP] Once the staging branch is up-to-date you can use the
+> [`create-release-proposal`][] action to generate the proposal.
+
+![screenshot running action via github ui](https://github.com/user-attachments/assets/244795d0-3753-45ea-a259-169d15c40239)
```

Comment:
Using the `gh` API can be an option; I just feel most people tend to use it via the UI. Regarding stability, we are using it on https://github.com/nodejs/node/blob/main/doc/contributing/security-release-process.md#post-release and it still works.

I have been using the GitHub self-hosted assets for a while, and I haven't encountered any issues so far.

Reply:
> I just feel most people tend to use it via the UI

That's quite probable indeed, however my point is that we should probably only document the CLI instead – which is unlikely to change anytime soon – rather than the web UI – which is more likely to change, and quite explanatory anyway.

Reply:
> That's quite probable indeed, however my point is that we should probably only document the CLI instead

👍 agreed, the release guide is very much cli-centric so it makes more sense to document that instead and anyone wanting the use the webapp UI will figure it out.

Reply:
PTAL 115e5d5